app/src/main/java/com/dragic/beggining_recyclerview_kodeco/ui/FavoritesFragment.kt

```diff
@@ -35,18 +35,48 @@ import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import com.dragic.beggining_recyclerview_kodeco.R
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.dragic.beggining_recyclerview_kodeco.databinding.FragmentFavoritesBinding
+import com.dragic.beggining_recyclerview_kodeco.model.CreatureStore
 
 
 class FavoritesFragment : Fragment() {
 
-  companion object {
-    fun newInstance(): FavoritesFragment {
-      return FavoritesFragment()
+    private lateinit var adapter: CreatureAdapter
+    private lateinit var binding: FragmentFavoritesBinding

```

Comment:
Same here for the adapter and binding :)

Reply:
done